installation/gcp/dm/components/roles.py

```diff
@@ -21,13 +21,10 @@
 def GenerateConfig(context):
     """Creates the custom roles"""
 
-    deployment_name_hash = sha1(context.env["deployment"].encode("utf8")).hexdigest()[:10]
```

Comment:
im not familiar fully with why we use hash here, but will this change impact any logic (name hitting the size limit)?